transcription/transcription_test.go

```diff
@@ -0,0 +1,55 @@
+package transcription
+
+import (
+	"errors"
+	"net/smtp" // mock
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
```

Comment:
`username, password, to` values here are dummy - is that ok or will that mean that email gets rejected?